libbeat/processors/add_cloud_metadata/provider_aws_ec2.go

```diff
@@ -0,0 +1,24 @@
+package actions
```

Comment:
Not related to this change, but why is this package called `actions`? @andrewkroh